templates/repo/commits_list_small.tmpl

```diff
@@ -33,18 +33,16 @@
 			{{else}}
 				<span class="{{$class}}">
 			{{end}}
-					<span class="shortsha">{{ShortSha .ID.String}}</span>
+			<span class="shortsha">{{ShortSha .ID.String}}</span>
 			{{if $.Issue.PullRequest.BaseRepo.Name}}
 				</a>
 			{{else}}
 				</span>
 			{{end}}
 		</span>
 
-		<span class="message-wrapper">
-			{{ $commitLink:= printf "%s/%s/%s/commit/%s" AppSubUrl  $.Issue.PullRequest.BaseRepo.OwnerName $.Issue.PullRequest.BaseRepo.Name .ID }}
-			<span class="mono commit-summary has-emoji{{if gt .ParentCount 1}} grey text{{end}}" title="{{.Summary}}">{{RenderCommitMessageLinkSubject .Message ($.Issue.PullRequest.BaseRepo.Link|Escape) $commitLink $.Issue.PullRequest.BaseRepo.ComposeMetas}}</span>
-		</span>
+		{{ $commitLink:= printf "%s/%s/%s/commit/%s" AppSubUrl  $.Issue.PullRequest.BaseRepo.OwnerName $.Issue.PullRequest.BaseRepo.Name .ID }}
+		<span class="mono commit-summary has-emoji{{if gt .ParentCount 1}} grey text{{end}}" title="{{.Summary}}">{{RenderCommitMessageLinkSubject .Message ($.Issue.PullRequest.BaseRepo.Link|Escape) $commitLink $.Issue.PullRequest.BaseRepo.ComposeMetas}}</span>
```

Comment:
```suggestion
		<span class="mono commit-summary {{if gt .ParentCount 1}} grey text{{end}}" title="{{.Summary}}">{{RenderCommitMessageLinkSubject .Message ($.Issue.PullRequest.BaseRepo.Link|Escape) $commitLink $.Issue.PullRequest.BaseRepo.ComposeMetas}}</span>
```

Might as well remove this unused class here (was previously used by JS emoji conversion). This looks to be the last occurence in the codebase.

Reply:
suck back in from code copied before that PR and committed after